### DISCUSSION TITLE
Applied Science: split Cooking and Science

### DISCUSSION THREAD

**Discussion by ghost** [cataclysmbn/Cataclysm-BN#4479] [General]
The following is going to be a summary of Discord discussion.

The goal is to split cooking and science to allow for smoother and more sensible progression, to make both skills more useful and effective as well as distinct from one another, and to overall provide greater balancing. A big part of this is going from 0 to 7 cooking via foraged ingredients + animal butchery products allows you to get into smokeless powder production without having trained science. I have a png compiled if interested on the easiest ways to get cooking for each level.

We would create a system of 5 tiers that are focused on methods and supercategories of recipes,
Cooking Tiers:
1: Mixing and pan frying, Boiling most things.
2: Stir fries, more complicated mixing/frying, fermentation
3: Deep frying, steaming, canning, distillation
4: Pasta, bread, cookies, Pizza, any "from scratch" basic tier baking/noodles, etc.
5: Pies, Cake. Complex baking which is easy to screw up for the uninitiated.

Science Tiers: 
1: Fermentation & Distillation via secondary skill, fuels AKA lamp oil, some electrical items
2. Acid, chemical reagents, possibly lasers/electric weapons and plasma via secondary skill,
3. Drug synthesis, complex chemistry. Toxicology and poison, possibly geology here or tier 2.
4. Explosives, EMP weaponry, possibly CBM stuff via secondaries,
5. Mutagenics

Additional notes: Geology requires a mining overhaul, toxicology requires extra relic data fields for ammunition items, triffids and mycus are candidates for use in science as ingredients to make chemicals. Bulk recipes are a good idea, especially if we call for obsoleted science equipment from DDA.

CV has a list of ideas with triffid loot, including "Triffid bud sap, triffid stinger powder, leech flower oil", as additional items and recipes.

This is now an open call for any feedback and additional ideas to create the best applied science skill possible.

**Comment by ghost** [comment]
Not mentioned in the initial post: It can be assumed we'd have to rework morale and focus as well as morale effects on all the food in the game and how long morale buffs last. That is an easier solution than Applied science because it's mainly changing the values around for how much and how long.

**Comment by DoubleBullfrog** [comment]
This would be a good area for something similar to the proficiencies system - wait, hear me out. Instead of adding new skills, having proficiencies for sub-aspects of skills like chemistry could allow for adding new aspects to crafting without needing to add new skills. I think having Cooking transition into the general chemistry skill is good design and makes Cooking not irrelevant later on in the game as well as smoothing the jump to making mutagens and drugs. So you'd get up to a high level of Cooking, then have to practice some amount of basic chemistry stuff to unlock the Chemistry proficiency, and then get full crafting speed for chemistry related stuff in the cooking skill.

Proficiencies should also provide some minor bonus to make them feel more rewarding rather than just being keys to unlock parts of the crafting system; Chemistry proficiency could provide an increase to batch-crafting drug recipes, a Mad Science proficiency could speed up batch-crafting mutagens, a Bionics proficiency for CBM crafting that provides bonus speed for disassembling CBMs and a small increase to CBM install chance, that sort of thing.
